bukuserver/forms.py

```diff
@@ -1,5 +1,6 @@
 """Forms module."""
 # pylint: disable=too-few-public-methods, missing-docstring
+from __future__ import annotations
```

Comment:
Workarounds like these should have a comment to indicate specific purpose; i.e.:
```py
from __future__ import annotations  # Python <3.9 may need this to handle typehints like tuple[...]
```